pywbem/cim_operations.py

```diff
@@ -1497,6 +1518,16 @@ def operation_recorder_stage_result(self, ret, exc):
             recorder.stage_pywbem_result(ret, exc)
             recorder.record_staged()
 
+    @classmethod
+    def future_logging_reset(cls):
+        """
+        Reset the activation of logging and the log detail levels for future
```

Comment:
This is a public method but is it intended for anything but our tests??

Reply:
Right now the user can only activate logging for future connections, but has no means to deactivate it again. It is just cleaner if we provide a means to do that.